bin/plotting/pycbc_page_foundmissed

```diff
@@ -200,7 +209,10 @@ if args.gradient_far:
         # Set up tick labels - there will always be 5
         min_tick = numpy.ceil(min(numpy.log10(color)))
         max_tick = numpy.floor(max(numpy.log10(color)))
-        tick_step = numpy.floor((max_tick - min_tick) / 5)
+        if (min_tick - max_tick) < 5:
```

Comment:
put a comment to explain what this does? Also does it invalidate the previous comment about 'there will always be 5' ?

Reply:
This was fixed in #4153, so was changed by a rebase anyway